benchmarks/Tosa/CMakeLists.txt

```diff
@@ -0,0 +1,164 @@
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+
+################################################################################
+#                                                                              #
+# Benchmark models for Tosa                                                    #
+#                                                                              #
+# Each module specification should be a list that contains the following       #
+# fields: MODULE_NAME, MODULE_TAGS, MLIR_SOURCE, ENTRY_FUNCTION,               #
+# FUNCTION_INPUTS. See iree_mlir_benchmark_suite definition for details about  #
+# these fields.                                                                #
+#                                                                              #
+################################################################################
+
+#set(MOBILENETSSD_FP32_MODULE
+#  "MobileNetSSD"                  # MODULE_NAME
+#  "fp32"                          # MODULE_TAGS
+#  "https://storage.googleapis.com/iree-model-artifacts/MobileNetSSD.tar.gz" # MLIR_SOURCE
+#  "main"                          # ENTRY_FUNCTION
+#  "1x320x320x3xf32"               # FUNCTION_INPUTS
+#)
+#
+#set(MOBILENETV1_FP32_MODULE
```

Comment:
I think we can drop MobileNetV1. It's quite old and there are already V2/V3 benchmarked.